### DISCUSSION TITLE
How can I know what datasets gs-quant provide?

### DISCUSSION THREAD

**Discussion by cheukhin1024** [goldmansachs/gs-quant#218] [Q&A]
It requires user login ( https://marquee.gs.com/l/?authLevel=40000&From=aHR0cHM6Ly9tYXJxdWVlLmdzLmNvbS9zL2RldmVsb3Blci9kYXRhc2V0cy9DQg%3D%3D ).

Can the public request data using gs-quant?

**Comment by chcr9095** [comment]
Hi in order to access data you are required to have a 'client ID' and 'client secret' from GS API. Certain datasets require premium access while others are free to users who have an existing GS sales rep or licensing agreement.
